code/go/0chain.net/smartcontract/zcnsc/test_factory_helpers.go

```diff
@@ -0,0 +1,173 @@
+package zcnsc
```

Comment:
If this file is only used for testing, use the `_test` suffix so it is not part of the production build.

Reply:
Moved to test module